docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # The base image (buildbox:latest) is built by running `make -C build.assets`
 # from the base repo directory $GOPATH/gravitational.com/teleport
-FROM quay.io/gravitational/teleport-buildbox:go1.14.4
+FROM quay.io/gravitational/teleport-buildbox:go1.15.3
```

Comment:
at least in Dockerfiles, this could be a configuration - e.g. via ARG:
```dockerfile
ARG GOVERSION=go1.15.3
FROM quay.io/gravitational/teleport-buildbox:${GOVERSION}
```

Reply:
I did try this originally, but then it added an extra layer of indirection trying to make sure that the correct version was always set for every way that the build target can be called. I think at least this way it's very obvious what version you're going to be building with.